server.go

```diff
@@ -212,8 +229,16 @@ func newServer(listenAddrs []string, notifier chainntnfs.ChainNotifier,
 		IDKey:    s.identityPriv.PubKey(),
 		Wallet:   wallet,
 		Notifier: s.chainNotifier,
-		SendToDiscovery: func(msg lnwire.Message) {
-			s.discoverSrv.ProcessLocalAnnouncement(msg,
+		SignNodeKey: func(nodeKey, fundingKey *btcec.PublicKey) (*btcec.Signature,
+			error) {
+			return fakeSig, nil
+		},
+		SignAnnouncement: func(msg lnwire.Message) (*btcec.Signature,
+			error) {
```

Comment:
In line with my comments above, both of these closures should be removed and replaced with a single `MessageSigner` interface that is fully hooked up and produces valid signatures.